automathor.py

```
@@ -0,0 +1,170 @@
+        if result.returncode != 0:
+            raise RuntimeError(f"Command failed with error: {result.stderr}")
+
+        context = Context(self.file, self.line, None, None, None, None, None, [])
+        for line in result.stdout.strip().split("\n"):
+            logging.debug(f"PROCESSING {line}")
+            data = self.parse_line(line)
+            if data.line_number == self.line:
+                context.commit = data.commit
+                context.author_email = data.author_email
+                context.datetime = data.datetime
+                context.todo = self.text.strip().lstrip("#:TODO ")
```

Ruff (0.8.2)

Using `.strip()` with multi-character strings is misleading

(B005)